From e92630c3e6ef2253b44429680910912671589680 Mon Sep 17 00:00:00 2001
From: Albert Uchytil <auchytil@redhat.com>
Date: Fri, 11 Apr 2014 13:24:30 +0200
Subject: [PATCH] Changed scl_prefix macro, that now accepts a parameter. Users
 can now easily override default scl_prefix in some cases. (#1057634)

---
 macros.scl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/macros.scl b/macros.scl
index 45ae422705517017eaa7a20d57a15f82495350ea..f4685137301ca3a979b71ac1a934134b666837a5 100644
--- a/macros.scl
+++ b/macros.scl
@@ -35,10 +35,11 @@ package or when debugging this package.
 %endif
 %{nil}}}
 
+%scl_prefix()   %{?scl:%(if [ "%1" = "%%1" ]; then echo "%{scl}-"; else echo "%1-"; fi)}%{!?scl:%{nil}}
+
 %scl_package() %{expand:%{!?_root_prefix:
 %global pkg_name		%1
 %global scl_name		%{scl}
-%global scl_prefix		%{scl}-
 %global scl_runtime		%{scl}-runtime
 %{!?scl_basedir:		%global scl_basedir /opt}
 %{!?scl_vendor:			%global scl_vendor rh}
-- 
1.9.0

